ksql-cli/src/main/java/io/confluent/ksql/cli/console/cmd/CliSpecificCommand.java

```diff
@@ -14,10 +14,34 @@
 
 package io.confluent.ksql.cli.console.cmd;
 
+import java.io.PrintWriter;
+import java.util.List;
+
 public interface CliSpecificCommand {
+
+  /**
```

Comment:
Thanks for adding javadocs 👍 helps me ramp up faster!

Reply:
No problem. :D